packages/component-utils/src/__demo__/components/Code.js

```diff
@@ -15,16 +15,12 @@
  */
 
 /* @flow */
-import { ThemeProvider } from '../../index';
-import Sample from '../components/Sample';
+import { createStyledComponent } from '../../index';
 
-export default {
-  title: 'Global theme var override',
-  description:
-    'This will affect all components inside this ThemeProvider that use this variable.',
-  scope: { ThemeProvider, Sample },
-  source: `
-    <ThemeProvider theme={{ color_primary: 'mediumvioletred' }}>
-      <Sample />
-    </ThemeProvider>`
-};
+export default createStyledComponent('code', ({ theme }) => ({
+  backgroundColor: theme.color_gray_20,
+  borderRadius: theme.borderRadius_1,
+  fontFamily: theme.fontFamily_monospace,
+  padding: theme.spacing_half,
+  whiteSpace: 'nowrap'
```

Comment:
The `whiteSpace: 'nowrap'` on the `code` component causes the contained text to extend off to the right of smaller screens.  Is it necessary?

Reply:
Relatedly, I noticed that it is the `flexbox` styles on the `header` element in the `ComponentDoc` that is causing the extra scroll width on small screens.

Reply:
Good catch; thanks. That issue is worse than fixing the wrapping within the prose, so I removed it.

The `header` issue will be addressed with #271.